build.sbt

```diff
@@ -2,379 +2,402 @@ import java.io._
 import scala.util.Try
 import scala.xml.{Node => XmlNode, NodeSeq => XmlNodeSeq, _}
 import scala.xml.transform.{RewriteRule, RuleTransformer}
+import org.scalajs.sbtplugin.ScalaJSCrossVersion
 import org.scalameta.os
 import PgpKeys._
 import UnidocKeys._
 import sbt.ScriptedPlugin._
 import com.trueaccord.scalapb.compiler.Version.scalapbVersion
 
-lazy val ScalaVersion = "2.11.8"
+lazy val ScalaVersion = sys.env.getOrElse("SCALA_VERSION", "2.11.8")
 lazy val ScalaVersions = Seq("2.11.8", "2.12.1")
 lazy val LibrarySeries = "1.7.0"
-lazy val LibraryVersion = sys.props.getOrElse("scalameta.version", computePreReleaseVersion(LibrarySeries))
+lazy val LibraryVersion =
+  sys.props.getOrElse("scalameta.version", computePreReleaseVersion(LibrarySeries))
 
 // ==========================================
 // Projects
 // ==========================================
 
-lazy val scalametaRoot = Project(
-  id = "scalametaRoot",
-  base = file(".")
-) settings (
-  sharedSettings,
-  noPublish,
-  unidocSettings,
-  commands += Command.command("ci-fast") { state =>
-    // NOTE. The so/wow/such/very commands are from sbt-doge and are used to
-    // run commands with the correct scalaVersion in each project.
-    "so scalametaRoot/test" ::
-    "so contrib/test" ::
-    // slow tests below
-    "much doc" ::
-    state
-  },
-  commands += Command.command("ci-slow") { state =>
-    "very scalahost/test:runMain scala.meta.tests.scalahost.converters.LotsOfProjects" ::
-    "such testkit/test:runMain scala.meta.testkit.ScalametaParserPropertyTest" ::
-    "scalahostSbt/test" ::
-    state
-  },
-  commands += Command.command("ci-publish") { state =>
-    "very publish" ::
+name := "scalametaRoot"
```

Comment:
I removed the `lazy val scalametaRoot` project to get automatic aggregation for all defined projects. If I had kept it, we would need to 
```scala
.aggregate(
ioJVM, ioJS,
inputsJVM, inputsJS,
...
)
```
and it would get worse with native crossbuilding.